.github/workflows/master.yml

```diff
@@ -13,17 +13,21 @@ jobs:
       with:
         path: DCSA-EBL
         submodules: recursive
+
     - name: Set up Java JDK
-      uses: actions/setup-java@v1.4.3
+      uses: actions/setup-java@v2
       with:
-        java-version: 11
+        distribution: 'temurin'
+        java-version: 17
+
     - name: Cache local Maven repository
       uses: actions/cache@v2
       with:
         path: ~/.m2/repository
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
```

Comment:
These two can be merged into one since `actions/setup-java@v2` by adding "cache: maven" to the `setup-java` configuration.

Reply:
Fixed this, though I am curious that only EBL seems to use the cache

Reply:
It would seem that this mutation does not have that a strong enough evolutionary advantage to outperform the less evolved strains of the CI species! Perhaps we need to assist the mutation by helping it spread to other CI instances so it is more likely to reproduce than the previous strain...